src/LibraryViewExtension/Handlers/NodeItemDataProvider.cs

```diff
@@ -74,9 +74,9 @@ protected Stream GetNodeItemDataStream(IEnumerable<NodeSearchElement> searchEntr
         protected virtual object CreateObjectForSerialization(IEnumerable<NodeSearchElement> searchEntries)
         {
             var data = new LoadedTypeData<LoadedTypeItem>();
-            data.loadedTypes = searchEntries
-                //.Where(e => !e.ElementType.HasFlag(ElementTypes.Packaged))
-                .Select(e => CreateLoadedTypeItem<LoadedTypeItem>(e)).ToList();
+
+            // Converting searchEntries to another list so as to avoid modifying the actual searchEntries list when iterating through it. 
```

Comment:
why was this condition removed?
`!e.ElementType.HasFlag(ElementTypes.Packaged))`

Reply:
Was it used before, it was commented out and not sure for what reason

Reply:
ahh sorry, overlooked that. 😢